docs/src/concrete-playback.md

```diff
@@ -0,0 +1,59 @@
+# Concretely playing back proof harnesses
+
+When the result of a certain check comes back as a `FAILURE`,
+this _experimental_ feature allows users to debug their proof harness and understand what
```

Comment:
you should probably say that Kani offers two different paths today.
 - Visualize: Kani provide an HTML text based trace that enumerate the execution steps that led to the property failure.
 - Concrete playback: This is a new _experimental_ feature that generates a unit test that when executed will trigger the failure seen during the verification.

Reply:
Done